src/App.js

```diff
@@ -1,13 +1,105 @@
 import React, { Component } from 'react'
+import PropTypes from 'prop-types'
 import { Instructions } from './components'
 import './styles/app.css'
 
+class Counter extends Component {
```

Comment:
Move this component to its own file. Don't keep multiple components in a single file.